charts/portworx/templates/portworx-k8s-secrets.yaml

```diff
@@ -1,8 +0,0 @@
-{{- $secretType := .Values.secretType | default "k8s" }}
```

Comment:
Why did we delete this?

Reply:
> Why did we delete this?

We don't need to turn every helm value into a variable. This same thing can be expressed like so:

```yaml
# values.yaml
secretType: k8s

# storageClass.yaml
{{ .Values.secretType }}
```

I don't understand why `$secretType` needs to be used when it can be defined in the values.yaml file and called using basic helm syntax.

Reply:
But if you check, here we are creating the Portworx namespace that should host the secret that the user wants to use, if we don't do it here, then we need to end up documenting that 
1. create a namespace called portworx
2. then create your secret that you want us to use in that namespace.
May be a better name for this variable will be `createSecretNamespace` and if that is set to true or we detect secret type as `k8s` we create the namespace.